src/test/resources/AggregatedDocumentInternalComposition.json

```diff
@@ -15,70 +23,94 @@
-                              }
-                           ]
+                            "submitter": {
+                                "name": "Jane Doe",
+                                "email": "jane.doe@company.com"
+                            },
+                            "SCSEventId": "3ce9df6e-cd45-4320-ae66-945f038caa1b",
+                            "svnIdentifier": null,
+                            "sourceCreations": [
+                                {
+                                    "SCCEventId": "ac085e24-ac4c-41be-912d-08c7afd32285",
+                                    "author": {
+                                        "name": "John Doe",
+                                        "id": "johndoe",
+                                        "email": "john.doe@company.com",
+                                        "group": "Team Gophers"
+                                    },
+                                    "issues": [
+                                    ]
+                                },
+                                {
+                                    "SCCEventId": "552ad6a4-c522-47e2-9195-0481930979e4",
+                                    "author": null,
+                                    "issues": [
+                                        {
+                                            "tracker": "JIRA",
+                                            "id": "JIRA-1234",
+                                            "type": "BUG",
+                                            "uri": "http://jira.company.com/browse/JIRA-1234",
+                                            "transition": "RESOLVED"
+                                        }
+                                    ]
+                                }
+                            ]
                         }
-                     ]
-                  }
-               ],
-               "name":"My composition",
-               "time":2000
-            }
-         ]
-      },
+                    ],
+                    "name": "My composition",
+                    "time": 2000
+                }
+            ],
+            "artifacts": [
+                {
+                    "id": "1100572b-c3j4-441e-abc9-b62f48080011"
+                }
+            ]
+        },
         "testCaseExecutions": [
             {
-                "testCaseTrigeredEventId": "6d3df0e0-404d-46ee-ab4f-3118457148f4",
-                "testCaseTrigeredTime": 1490777327230,
+                "testCaseTriggeredEventId": "6d3df0e0-404d-46ee-ab4f-3118457148f4",
+                "testCaseTriggeredTime": 1490777327230,
                 "testCaseFinishEventId": "11109351-41e0-474a-bc1c-f6e81e58a1c9",
                 "testCaseStartedTime": 1481875925916,
                 "testCaseStartedEventId": "cb9d64b0-a6e9-4419-8b5d-a650c27c59ca",
                 "testCaseFinishedTime": 1481875935919,
-                "testCaseDuration": 10003,
```

Comment:
why has this been removed? It should be here. This is the test for process rules